config/ports.env

```diff
@@ -1,6 +1,4 @@
-METADATA_PORT=55500
-METADATA_URL=http://localhost:55500/dev.xml
-
+METADATA_PORT=80
```

Comment:
So METADATA in verify-local-startup is actually an Nginx instance and I've not had much luck getting it to run on anything other than port 80... So internally within the docker network it runs on 80, externally you can access it directly on port 55500 to see the XML and grab the certs for debugging.

Reply:
Yep, agreed. As is though this variable doesn't get used - in the docker compose the internal port is hard coded to port 80. So I think it might be better to keep this var as 55500 and use it instead of the new var created.